spring-cloud-kubernetes-discovery/src/main/java/org/springframework/cloud/kubernetes/discovery/KubernetesReactiveDiscoveryClient.java

```diff
@@ -38,14 +38,14 @@ public KubernetesReactiveDiscoveryClient(WebClient.Builder webClientBuilder,
 		if (!StringUtils.hasText(properties.getDiscoveryServerUrl())) {
 			throw new DiscoveryServerUrlInvalidException();
 		}
-		this.webClient = webClientBuilder.baseUrl(properties.getDiscoveryServerUrl()).build();
```

Comment:
minor things, just removed `this.`, I am not sure I am irritated by this default value :)